configuration/api.go

```diff
@@ -139,6 +139,76 @@ func (a *API) GetBot(id string, fields []string) (*Bot, error) {
 	return resp, err
 }
 
+// ResetBotSecret resets value of secret used to get bot token.
+func (a *API) ResetBotSecret(id, ownerClientID string) (string, error) {
+	var resp resetBotSecretResponse
+	err := a.Call("reset_bot_secret", &resetBotSecretRequest{
+		BotID:         id,
+		OwnerClientID: ownerClientID,
+	}, &resp)
+
+	return resp.Secret, err
+}
+
+// IssueBotToken issues a token for bot with given ID.
+func (a *API) IssueBotToken(botID, botSecret, organizationID, clientID string) (string, error) {
+	var resp issueBotTokenResponse
+	err := a.Call("issue_bot_token", &issueBotTokenRequest{
+		BotID:          botID,
+		BotSecret:      botSecret,
+		OrganizationID: organizationID,
+		ClientID:       clientID,
+	}, resp)
+	return resp.Token, err
+}
+
+// CreateBotTemplate allows to create bot template and returns its' ID.
```

Comment:
```suggestion
// CreateBotTemplate allows to create bot template and returns it's ID.
```

also it should return it's secret

Reply:
also pls adjust `CreateBot` to return secret